runblender.ipynb

```diff
@@ -59,9 +58,8 @@
         "colab": {}
       },
       "source": [
-        "if use_drive:\n",
-        "  from google.colab import drive\n",
-        "  drive.mount('/content/drive')\n"
```

Comment:
I would keep this as there are people who chose "Run all Cells" just as a sanity check.